mina-daemon/scripts/generate-libp2p-keys.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+if [ -z "$MINA_LIBP2P_PRIV_KEYPATH" ]; then
+    echo "The MINA_LIBP2P_PRIV_KEYPATH environment variable is not set or is empty."
+    exit 1
+fi
+
+# Function to generate a random password
+generate_password() {
+    password=$(openssl rand -base64 64 | tr -dc 'a-zA-Z0-9' | head -c 64)
```

Comment:
This line generates a 64 character password using only digits, lowercase and uppercase. I have followed the same pattern as the ITN community generated keys